source/_integrations/lutron.markdown

```diff
@@ -73,3 +73,18 @@ After setup, scenes will appear in Home Assistant using the area, keypad and but
 ## Occupancy Sensors
 
 Any configured Powr Savr occuancy sensors will be added as occupancy binary sensors. Lutron reports occupancy for an area, rather than reporting individual sensors. Sensitivity and timeouts are controlled on the sensors themselves, not in software.
+
+## Example Automations
```

Comment:
Add an empty line after a header.

Reply:
check the latest commit, missed this in the conventions. thanks for the review.